packages/cra-template-mephisto-review/template/src/GridView.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { Card, Elevation, H4 } from "@blueprintjs/core";
+
+function GridView({ data }) {
+  const history = useHistory();
+
+  return data && data.length > 0 ? (
+    <div className="grid-container">
+      {data.map((item) => (
+        <Card
+          key={item.id}
+          interactive={true}
+          elevation={Elevation.TWO}
+          className="grid-item"
+          onClick={() => history.push(`/${item.id}`)}
+        >
+          <div className="grid-item-text">
+            <pre>{item.data}</pre>
```

Comment:
Looking at the screenshot you provided, I see that the whole payload is wrapped in a string " ". Can you look into ways to avoid those quotes? e.g. I think the JSON itself should be embedded, not the string representation of the JSON